### PR TITLE
[Wf-Diagnostics] Set query handler for diagnostics workflow to provide result

### PR DESCRIPTION
**Pull request by sankari165**
**What changed?**
Diagnostics workflow has a query handler that provides the diagnostics result

**Why?**
Diagnostics workflow start will be triggered via api/cli and result will be fetched via query

**How did you test it?**
unit tests

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Review by dkrotx — APPROVED** [review]

File: service/worker/diagnostics/workflow.go [hunk 1]
```diff
@@ -33,8 +33,9 @@ import (
 )
 
 const (
-	diagnosticsWorkflow = "diagnostics-workflow"
-	tasklist            = "wf-diagnostics"
+	diagnosticsWorkflow    = "diagnostics-workflow"
+	tasklist               = "wf-diagnostics"
```
Comment (ketsiambaku) [inline service/worker/diagnostics/workflow.go:37]: nit: I'd name it maybe "diagnostics-tasklist" or something similar for consistency in names

File: service/worker/diagnostics/workflow.go [hunk 2]
```diff
@@ -54,6 +55,14 @@ type DiagnosticsWorkflowResult struct {
 }
 
 func (w *dw) DiagnosticsWorkflow(ctx workflow.Context, params DiagnosticsWorkflowInput) (DiagnosticsWorkflowResult, error) {
+	var result DiagnosticsWorkflowResult
+	err := workflow.SetQueryHandler(ctx, queryDiagnosticsReport, func() (DiagnosticsWorkflowResult, error) {
+		return result, nil
+	})
+	if err != nil {
+		return result, err
```
[the remark this answers is not in the document]
Comment (ketsiambaku) [reply service/worker/diagnostics/workflow.go:63]: +1 pointer would be even better because `return nil, err` is much clearer

**Review by ketsiambaku — APPROVED** [review]

File: service/worker/diagnostics/workflow.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (sankari165) [reply service/worker/diagnostics/workflow.go]: updated

File: service/worker/diagnostics/workflow.go (hunk 1 above)
Comment (sankari165) [reply service/worker/diagnostics/workflow.go]: done

**Pull request merged**